build.gradle

```diff
@@ -91,15 +91,20 @@ allprojects {
     apply plugin: 'base'
 
     afterEvaluate { project ->
+        // format is '{project.group}.{project.name}'.
+        // note: project.group returns a period delimited the fully qualified package name.
         def composeDeps = [
-                "airbyte-config:init",
```

Comment:
anything that had a `:` wasn't working in this because project.name just returns the name no prefix (e.g. init).